build/release/release.sh

```diff
@@ -14,6 +14,7 @@ case ${action} in
         ;;
 
     publish.init)
+        publish_version_file
```

Comment:
we're now doing this for the publish target and not just the promote target...any potentially unintended side effect you can think of?